LCM/scripts/RemoveModule.py

```diff
@@ -50,6 +50,37 @@ def RemoveMirroredDirectory(moduleDir, omiDir):
 
 print("Removing module " + moduleName)
 
+# special section for nxOMSAutomationWorker module
+# the Linux Hybrid worker needs to be killed before the module is removed
+# also a good idea to remove the state.conf file
+# we preserve the worker.conf file in case the user wants to preserve his settings
```

Comment:
worker.conf may have some user modified configuration that might be needed when the module gets installed again. RemoveModule is a soft operation that should be easily reversible by InstallModule. OMS agent remove/purge will get rid of all .conf such files if a hard removal is required. Updated the comments in the file to reflect this change